web/impact/requirements/base.txt

```diff
@@ -36,7 +36,7 @@ djangorestframework
 djangorestframework-jwt
 drf-schema-adapter
 drf-tracking
-git+https://github.com/masschallenge/django-accelerator.git@2017.11.09#egg=django-accelerator
+git+https://github.com/masschallenge/django-accelerator.git@AC-5038-njw#egg=django-accelerator
```

Comment:
Note that this is different from the version in AC-5038-mcproject (plus it is not a release)